pkg/utils/file/file.go

```diff
@@ -26,10 +26,14 @@ import (
 	"path/filepath"
 )
 
-func CreateDirIfNotExists(dir string) (err error) {
-	if err := os.MkdirAll(dir, 0755); err != nil && !os.IsExist(err) {
-		return err
+// EnsureDir ensures the directory exists.
```

Comment:
we should also clarify that the function will create dir if not exist in the comment.